guides/TAGS.md

```diff
@@ -4,15 +4,15 @@ Tagging a release
 In order to tag a release you first need to checkout the branch you aim to tag (usually `master`)
 
 * Run `git tag -s <tag> -m <message>`
-	* `<tag>` is the same as the branch name (e.g. `release/1.7.3` or `beta/1.0.0`)
+	* `<tag>` is the semver value in the release branch name (e.g. `v1.7.0`, see [version control](VERSIONING.md))
 	* `-s` ensures the tag is signed
 	* provide a message to describe the changes at a high level
 * `git push —-tags`
```

Comment:
The following is a safer command:
```suggestion
* `git push --follow-tags`
```
It will only push _relevant_ tags (as in tags pointing to the branch HEAD you are working with) rather than all local tags.  It also solves the issue whereby pushing tags before pushing the merge into `master` to the `remote` can cause the tag on the remote to point to the old `HEAD` of `master` (i.e. the previous release).

It might also be useful to add a note about this reasoning to the docs.